src/views/dialogs/DepositDialog2/RouteOptions.tsx

```diff
@@ -0,0 +1,123 @@
+import { RouteResponse } from '@skip-go/client';
+import { formatUnits } from 'viem';
+
+import { LightningIcon, ShieldIcon } from '@/icons';
+
+import { DepositSpeed } from './types';
+
+type Props = {
+  routes: { slow?: RouteResponse; fast?: RouteResponse };
+  isLoading: boolean;
+  disabled: boolean;
+  selectedSpeed: DepositSpeed;
+  onSelectSpeed: (route: DepositSpeed) => void;
+};
+
+export const RouteOptions = ({
+  routes,
+  isLoading,
+  selectedSpeed,
+  onSelectSpeed,
+  disabled,
+}: Props) => {
+  // TODO(deposit2.0): finalize error handling here
+  if (!routes.slow && !routes.fast) {
+    return <div>There was a error loading deposit quotes.</div>;
+  }
+
+  const fastOperationFee = // @ts-ignore
+    routes.fast?.operations.find((op) => Boolean(op.goFastTransfer))?.goFastTransfer?.fee;
+  const totalFastFee = fastOperationFee
+    ? formatUnits(
+        BigInt(fastOperationFee.bpsFeeAmount ?? 0) +
+          BigInt(fastOperationFee.destinationChainFeeAmount ?? 0) +
+          BigInt(fastOperationFee.sourceChainFeeAmount ?? 0),
+        6
+      )
+    : '-';
+
+  return (
+    <div tw="flex gap-1">
+      <button
```

Comment:
Should we move this `<RouteButton>` into a re-usable component within `DepositDialog2` directory?
Inline styling -> Styled-components in future iteration?

Reply:
can def make a reusable component! ill keep it in the same file for now since its the only place being used :3